example-fileupload/README.md

```diff
@@ -0,0 +1,507 @@
+
+
+# Appwrite + VueJS =❤️
+This example is to showcase [Appwrite's JS API](https://github.com/appwrite/sdk-for-js) with [VueJS](https://vuejs.org/) by creating a File Uploading Application where you can Upload files and See recently uploaded files.
+
+## Prerequisites
+
+-   A Recent Version of NodeJS
+-   Yarn (Feel free to use NPM if you want to, just switch out the Yarn Commands for their NPM counterparts)
+-   [A locally running appwrite instance](https://appwrite.io/docs/installation).
+
+## Getting Started
+To get started as fast as possible we will use the Vue CLI, Here we will use NPX to launch the Vue CLI without installing it globally.
+``` shell
+npx @vue/cli create --default example-fileupload
+cd example-fileupload
+```
+that should create an app with VueJS 2, Babel and ESLint, while we are in the terminal we might as well install the appwrite SDK for later using:
```

Comment:
`that` should start with an uppercase letter, also Appwrite should start with an uppercase letter